client/src/App.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import Searchbar from "./components/Searchbar.jsx";
+import SearchDropdown from "./components/SearchDropdown.jsx";
+import CategoryDropdown from "./components/CategoryDropdown.jsx";
+import Navbar from "./components/Navbar.jsx";
+import bodyImg from "./images/bodyImg.png";
+
+const axios = require('axios');
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      cats: [],
+      catName: 'walnut',
+      searchDrop: false,
+      categoryDrop: false
+    };
+    this.getCat = this.getCat.bind(this);
+    this.onCatNameChange = this.onCatNameChange.bind(this);
+    this.getSearchedCat = this.getSearchedCat.bind(this);
+    this.searchDropAnimation = this.searchDropAnimation.bind(this);
+    this.searchDropFade = this.searchDropFade.bind(this);
+    this.categoryDropAnimation = this.categoryDropAnimation.bind(this);
+    this.categoryDropFade = this.categoryDropFade.bind(this);
+    this.getSuggestedCat = this.getSuggestedCat.bind(this);
+
+  }
+
+  componentDidMount () {
+    this.getCat();
+    console.log('component mounts')
+
+  }
+  getCat () {
+    // var catName = this.state.catName;
+    console.log('get request cat name: ');
+    axios.get(`/api/search/${this.state.catName}`)
+      .then(res => {
+        this.setState({
+          cats: res.data
+        })
+      })
+      .then(res => {
+        console.log(res)
+      })
+      .catch(err => {
+        console.log('axios error getting cats: ', err);
+      })
+  }
+
+  getSearchedCat (event) {
```

Comment:
This function and the one below it could be combined. The only difference is the context that they are called. So you can just pass the function to the needed location.